### PR TITLE
docs(guides): added config for using kayenta with Minio

### PR DESCRIPTION
**Pull request by e96wic**

### CODE REVIEW

File: setup/canary/index.md [hunk 1]
```diff
@@ -350,3 +350,33 @@ hal config canary prometheus account get
...
+
+## Set up Minio as storage provider
+Configure your canary analysis to use Prometheus as your telemetry provider.
+
+### Add a custom halyard profile for kayenta
+```
+  aws:
+    enabled: true
+    accounts:
+    - name: <name>
```
Comment (lwander) [inline setup/canary/index.md:365]: Is this only possible using a profile, not using the halyard commands? I'm all for showing both, but for consistency with the rest of the doc we should probably show the hal commands. 

@duftler

Comment (e96wic) [reply setup/canary/index.md:365]: Right now there is no option in halyard to specify the endpoint which is important for this to work. You can specify this for the front50 config but not yet for the kayenta config.

Comment (duftler) [reply setup/canary/index.md:365]: Ah, hadn't realized that. I'll have a look and add it the necessary option to halyard. Let's leave this open for the moment while I dig into it.

**Comment by duftler** [comment]
Adds support for `endpoint`: https://github.com/spinnaker/halyard/pull/968

**Comment by cfieber** [comment]
@duftler is this mergeable?

**Comment by duftler** [comment]
With the endpoint support in halyard, I don't think this is still necessary. Closing.

**Pull request closed without merging**